source/jormungandr/jormungandr/api.py

```diff
@@ -74,7 +74,11 @@ def access_log(response, *args, **kwargs):
         'status': response.status_code,
         'external_request_id': request.headers.get('x-request-id'),
     }
-    logger.info(u'"%(method)s %(path)s?%(query_string)s" %(status)s', d, extra=d)
+    if 'readyness' not in request.path:
+        logger.info(u'"%(method)s %(path)s?%(query_string)s" %(status)s', d, extra=d)
+    else:
+        logging.getLogger(__name__).debug(u'"%(method)s %(path)s?%(query_string)s" %(status)s', d, extra=d)
```

Comment:
is there a reason for not using `logger` here as above ?
```suggestion
        logger.debug(u'"%(method)s %(path)s?%(query_string)s" %(status)s', d, extra=d)
```

Reply:
Will test this this suggestion.

Reply:
This doesn't work as the logger is initialized with level = INFO.

Reply:
But if you change the logger's conf of `jormungandr.access` to `DEBUG` then it should work, no?
https://github.com/hove-io/navitia/blob/dev/source/jormungandr/jormungandr/default_settings.py#L96
And it's what we want I think (only log this when `jormungandr.access` conf is in DEBUG).

Reply:
After discussion: it would be cleaner to separate and route/activate at will.
But it's a bit more work and the use is not clear, so not spending the time for that and see in the future if it's needed.